integ-test/src/test/java/org/opensearch/sql/ppl/StandaloneIT.java

```diff
@@ -241,7 +241,7 @@ public QueryPlanFactory queryPlanFactory(ExecutionEngine executionEngine,
               new ExpressionAnalyzer(functionRepository), dataSourceService, functionRepository);
       Planner planner = new Planner(LogicalPlanOptimizer.create());
       QueryService queryService = new QueryService(analyzer, executionEngine, planner);
-      return new QueryPlanFactory(queryService, planSerializer);
+      return new QueryPlanFactory(queryService);
```

Comment:
Should we update `queryPlanFactory` here and in the plugin to remove the second argument? Probably you can remove function which `@Provides` a `PlanSerializer` instance too.